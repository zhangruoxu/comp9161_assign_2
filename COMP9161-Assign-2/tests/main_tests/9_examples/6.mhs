main = (letfun f x = x) 1;
