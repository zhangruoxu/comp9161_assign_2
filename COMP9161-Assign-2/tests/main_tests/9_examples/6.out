main :: Int = (letfun f :: (Int -> Int) x = x) 1;
